src/scope/network/ssh/ssh.js

```diff
@@ -160,11 +177,10 @@ export default class SSH implements Network {
     return this.exec('_list').then((str: string) => {
       const { payload, headers } = this._unpack(str);
       checkVersionCompatibility(headers.version);
-      return rejectNils(
-        payload.map((c) => {
-          return c ? ConsumerComponent.fromString(c) : null;
-        })
-      );
+      const consumerComponents = payload.map((c) => {
```

Comment:
Why did you change it?

Reply:
my mistake, this was the change for showing deprecate component with a flag,
i reverted the change